azure-pipelines.yml

```
@@ -3,7 +3,7 @@ variables:
   PreviewNumber: $[counter(variables['CurrentSemanticVersionBase'], 1001)]
   CurrentSemanticVersion: '$(CurrentSemanticVersionBase)-preview$(PreviewNumber)'
   NugetPackageVersion: '$(CurrentSemanticVersion)'
-  NET_VERSION: '8.0.x'
+  NET_VERSION: '9.0.100-rc.2.24474.11'
```

Change this to 9.0.x before merging

```suggestion
  NET_VERSION: '9.0.x'
```